examples/input/mouse_input_events.rs

```diff
@@ -3,6 +3,7 @@ use bevy::{
     prelude::*,
     window::CursorMoved,
 };
+use bevy_input::mouse::MouseWheel;
```

Comment:
Should this be
`input::mouse::{MouseButtonInput, MouseMotion, MouseWheel},`

Reply:
(line 2)

Reply:
Ah, yes, that would probably make more sense.  Rust Analyzer and it's auto-imports have spoiled me...sigh.